packages/build/src/log/logger.ts

```diff
@@ -190,7 +190,7 @@ export const getSystemLogger = function (
   return (...args) => fileDescriptor.write(`${reduceLogLines(args)}\n`)
 }
 
-export const addOutputGate = (logs: Logs, outputFlusher: OutputFlusher): Logs => {
+export const addOutputFlusher = (logs: Logs, outputFlusher: OutputFlusher): Logs => {
```

Comment:
Just renaming the method here to be consistent with the naming we ended up choosing. No functional change.